population/default_dataset_metadata.go

```diff
@@ -0,0 +1,73 @@
+package population
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/ONSdigital/dp-api-clients-go/v2/clientlog"
+	dperrors "github.com/ONSdigital/dp-api-clients-go/v2/errors"
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/pkg/errors"
+)
+
+type GetMetaDataInput struct {
+	AuthTokens
+	PopulationType string
+}
+
+type GetMetadataResponse struct {
+	PopulationType   string `json:"population-type"`
+	DefaultDatasetID string `json:"default_dataset_id"`
+	Edition          string `json:"edition"`
+	Version          int    `json:"version"`
+}
+
+func (c *Client) GetDefaultDatasetMetadata(ctx context.Context, input GetMetaDataInput) (GetMetadataResponse, error) {
```

Comment:
We're not getting metadata about the default dataset, we're getting metadata about the population type, which includes the id of the default dataset. This function would be bette called GetPopulationTypeMetadata, and the Inputas/Responses should be the same